frontend/src/types/emotion.d.ts

```diff
@@ -0,0 +1,7 @@
+import "@emotion/react";
+import type { MantineTheme } from "@mantine/core";
+
+declare module "@emotion/react" {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
Not sure if we can get around this - this is the exact example given by the Mantine docs (without the lint error)